docs/_static/js/docversion.js

```diff
@@ -1,17 +1,17 @@
 function setVersion(){
-        let doc = window.location.pathname.match(/^\/(api\/.*)$/) || window.location.pathname.match(/^\/versions\/[^*]+\/(api\/.*)$/);
+        let doc = window.location.pathname.match(/^\/versions\/[0-9.master]+\/([^*]+.*)$/);
```

Comment:
Wouldn't this fail if the version user is looking at is master, where `/versions/` is missing from the URL?

Reply:
Yep... good catch... I was testing the other side of the use case. 

I've been thinking about a related point. I could just force all the default traffic to /versions/master. This would solve a variety of thorny issues. Also, help with analytics. WDYT about that?

Reply:
Yeah I think I'd personally like to see `/versions/xyz` in the URL all the time.  In the future if we decide to change default to latest release rather than master, we can easily change the redirect and everything else should work as expected.